python-sdk/docs/concepts.rst

```diff
@@ -106,3 +106,58 @@ We also resolve the patterns in file path based on the :ref:`file_location`
 #. **Local** - Resolves ``File.path`` using the glob standard library (https://docs.python.org/3/library/glob.html)
 #. **S3** - Resolves ``File.path`` using AWS S3 prefix rules (https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-prefixes.html)
 #. **GCS** - Resolves ``File.path`` using Google Cloud Storage (GCS) wildcard rules (https://cloud.google.com/storage/docs/gsutil/addlhelp/WildcardNames)
+
+.. _dataset:
+
+Datasets
+--------
+
+An Airflow dataset is a stand-in for a logical grouping of data. Datasets may be updated by upstream "producer" tasks, and dataset updates contribute to scheduling downstream "consumer" DAGs.
+
+A dataset is defined by a Uniform Resource Identifier (URI):
+
+.. code-block:: python
+
+    from airflow import Dataset
+
+    example_dataset = Dataset("s3://dataset-bucket/example.csv")
+
+Astro SDK uses following URIs as datasets across its operators.
+
+#. :ref:`table` as a Dataset
+
+    .. literalinclude:: ../example_dags/example_datasets.py
+           :language: python
+           :start-after: [START dataset_table]
+           :end-before: [END dataset_table]
+
+#. File as a Dataset
+
+    .. literalinclude:: ../example_dags/example_datasets.py
+           :language: python
+           :start-after: [START dataset_file]
+           :end-before: [END dataset_file]
+
+Following examples show how to produce and use datasets for scheduling.
+
+#. **Dataset Producer** - Produces output dataset ``imdb_movies_table`` as a Table.
+
+    .. literalinclude:: ../example_dags/example_datasets.py
+           :language: python
+           :start-after: [START dataset_producer]
+           :end-before: [END dataset_producer]
+
+#. **Dataset Consumer** - DAG gets scheduled post input dataset Table ``imdb_movies_table`` is produced by the upstream DAG above.
+
+    .. literalinclude:: ../example_dags/example_datasets.py
+           :language: python
+           :start-after: [START dataset_consumer]
+           :end-before: [END dataset_consumer]
+
+Python SDK uses default datasets for its operators and the details of the default datasets generated and used by them can be found in the respective :ref:`operators` document.
```

Comment:
This ref is not being shown correctly on the docs.Can you take a look again.

<img width="1019" alt="Screenshot 2022-09-15 at 3 34 25 PM" src="https://user-images.githubusercontent.com/94376113/190377015-b2ea2a7e-0cdc-4f00-baba-45f003f133ff.png">

Reply:
What is the issue you're seeing. To me it is showing correctly. Also, clicking on the link takes to the appropriate page.

![Uploading Screenshot 2022-09-15 at 3.47.47 PM.png…]()

Reply:
Please ignore. I was looking the wrong way.

Reply:
Please ignore. I was checking using View file on the commit